docs/docs/train-tutorial.md

```diff
@@ -0,0 +1,338 @@
+)
+
+train_model(train_config=train_config, video_loader_config=video_loader_config)
+
+```
+
+To specify various parameters when running `train_model`, the first step is to instantiate [`TrainConfig`](configurations.md#training-arguments) and [`VideoLoaderConfig`](configurations.md#video-loading-arguments) with any specifications for model training and video loading respectively. The only two arguments that can be specified in `train_model` are `train_config` and `video_loader_config`.
+
+### Required arguments
+
+To run `train_model` in Python, you must specify both `data_directory` and `labels` when `TrainConfig` is instantiated.
+
+* **`data_directory (DirectoryPath)`:** Path to the folder containing your videos.
+
+* **`labels (FilePath or pd.DataFrame)`:** Either the path to a CSV file with labels for training, or a dataframe of the training labels. There must be columns for `filename` and `label`.
+
+In the command line, video loading configurations are loaded by default based on the model being used. This is not the case in Python. There are additional requirements for `VideoLoaderConfig` based on the model you are using.
+
+* **`video_height (int)`, `video_width (int)`:** Dimensions for resizing videos as they are loaded. 
+    - `time_distributed` or `european`: The suggested dimensions are 224x224, but any integers are acceptable
+    - `slowfast`: Both must be greater than or equal to 200
+* **`total_frames (int)`:** The number of frames to select from each video and use during training. 
+    * `time_distributed` or `european`: Must be 16
+    * `slowfast`: Must be 32
+
+The full recommended `VideoLoaderConfig` for the `time_distributed` or `european` model is:
+```python
+from zamba_algorithms.data.video import VideoLoaderConfig
+from zamba.models.megadetector_lite_yolox import MegadetectorLiteYoloXConfig
+
+megadetector_config = MegadetectorLiteYoloXConfig(
+    confidence=0.25, fill_mode="score_sorted", n_frames=16
+)
+video_loader_config = VideoLoaderConfig(
+    video_height=224,
+    video_width=224,
+    crop_bottom_pixels=50,
+    ensure_total_frames=True,
+    megadetector_list_config=megadetector_config,
+    total_frames=16,
+)
+```
+
+You can see the full default configuration for each model in `models/config`<!-- TODO: add link to source and update if needed><!-->. For detailed explanations of all possible configuration arguments, see [All Optional Arguments](configurations.md).
+
+## Default behavior
+
+By default, the model will be saved to a folder in the current working directory called `zamba_{model_name}`. For example, a model finetuned from the provided `time_distributed` model (the default) will be saved in `zamba_time_distributed`. 
+
+```console
+$ zamba train --data-dir example_vids/ --labels example_labels.csv
+$ ls zamba_time_distributed
+configuration.yaml 
+hparams.yaml
+time_distributed.ckpt
+events.out.tfevents.1632250686.ip-172-31-15-179.14229.0
+```
+
+`zamba_time_distributed` contains three files:
```

Comment:
I think we have more files now:

model checkpoint (``model_name``.ckpt), training configuration (configuration.yaml), Tensorboard logs (events.out.tfevents...), test metrics (test_metrics.json), validation metrics (val_metrics.json), and model hyperparameters (hparams.yml).

Reply:
Good catch. Are these descriptions right?

* `test_metrics.json`: The model's performance on the test subset
* `val_metrics.json`: The model's performance on the validation subset

Reply:
looks good to me!